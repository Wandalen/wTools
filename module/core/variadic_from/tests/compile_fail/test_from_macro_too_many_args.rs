// tests/compile_fail/test_from_macro_too_many_args.rs

#[ allow( dead_code ) ]
fn test_from_macro_too_many_args()
{
  let _ = variadic_from::from!( 1, 2, 3, 4 );
}