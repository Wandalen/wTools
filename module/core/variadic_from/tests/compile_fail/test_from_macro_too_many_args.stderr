error: Too many arguments
 --> tests/compile_fail/test_from_macro_too_many_args.rs:6:11
  |
6 |   let _ = variadic_from::from!( 1, 2, 3, 4 );
  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `variadic_from::from` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/compile_fail/test_from_macro_too_many_args.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/compile_fail/test_from_macro_too_many_args.rs`
